modules/prediction/prediction_component.cc

```diff
@@ -133,6 +133,16 @@ bool PredictionComponent::ContainerSubmoduleProcess(
     auto trajectory_msg = *ptr_trajectory_msg;
     MessageProcess::OnPlanning(trajectory_msg);
   }
+
+  // Read storytelling message and call OnStorytelling to update the
+  // StoryTellingContainer
+  storytelling_reader_->Observe();
+  auto ptr_storytelling_msg = storytelling_reader_->GetLatestObserved();
+  if (ptr_storytelling_msg != nullptr) {
+    auto storytelling_msg = *ptr_storytelling_msg;
+    MessageProcess::OnStoryTelling(storytelling_msg);
```

Comment:
So you make an unnecessary copy here. Use `const auto&`, or just call `OnStoryTelling(*ptr_storytelling_msg)`.